src/colvar/GHBFIX.cpp

```diff
@@ -154,7 +157,11 @@ GHBFIX::GHBFIX(const ActionOptions&ao):
   etafile.open(params);
   std::string it,jt;
   double eta;
-  while(etafile.scanField("itype",it).scanField("jtype",jt).scanField("eta",eta).scanField()) {
+  etafile.scanFieldList(list);
+  plumed_assert(list.size()==3)<<"unexpected size of custom fields in the file";
+
+  while(etafile.scanField(list[0],it).scanField(list[1],jt).scanField(list[2],eta).scanField()) {
+    plumed_assert(list[0]!=list[1] && list[0]!=list[2] &&  list[1]!=list[2])<<"headers in inputfile are not unique";
```

Comment:
@ThorbenF what you are doing here is basically meant to ignore the name of the field. However, one should expect the file to work equally well once you swap columns. So maybe it is better to assume names. For instance, one possibility would be to make sure that two of the names are equal to the name of the field in the other file with concatenated `_1` and `_2`. Would this make sense?